lib/beaker/hypervisor/vagrant.rb

```diff
@@ -161,6 +161,17 @@ def cleanup
       FileUtils.rm_rf(@vagrant_path)
     end
 
+    #snapshotting depends on https://github.com/scalefactory/vagrant-multiprovider-snap
```

Comment:
I know that the rest of this file isn't properly yard doc-ed, but please include yard docs for any new methods.